src/front/js/component/Pricing.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import "../../styles/Pricing.css";
+
+const Home = () => {
+  return (
+    <React.Fragment>
+      <section className="content-container">
+        <div className="columns">
+          <ul className="price">
+            <li className="col-header">Basic</li>
+            <li className="grey">9.99 €/ year</li>
+            <li>10 message to babbysitters</li>
+            <li>1 child</li>
+            <li>Full babysitter´s details </li>
+            <li className="grey">
+              <button className="button">Sign Up</button>
+            </li>
+          </ul>
+        </div>
+
+        <div className="columns">
+          <ul className="price">
+            <li className="col-header" style={{ backgroundColor: "#f06e26ff" }}>
```

Comment:
Inline styling? I'm tempted to not say something about it hahahah

I think that in this case is okay to do it this way, but let's always be careful with the size and frequency of those, okay?

But yeah, nothing to change in here 😌

Reply:
+1, please remove the style from html and include in our amazing CSS files!